docs/commands/states.md

```diff
@@ -0,0 +1,300 @@
+---
+title: Movement Detections
+category: Techniques
+mention:
+    - BedrockCommands
+    - zheaEvyline
+---
+
+## Introduction
+
+[Sourced By Bedrock Commands Community Discord](https://discord.gg/SYstTYx5G5)
+
+[Image Credits: Official Minecraft Wiki  -  ](https://minecraft.wiki/)__[CC BY-NC-SA 3.0](https://creativecommons.org/licenses/by-nc-sa/3.0/)__
+
+These command-techniques allow you to detect certain player/entity 'states' and subsequently execute your desired commands.
+
+> Note: For a more performant and high-accuracy approach, it is advisable to use [Animation Controllers](/animation-controllers/animation-controllers-intro)
+
+## is_moving
+
+This technique allows you to detect when your target is/isn't moving, accounting for even subtle movements.
+
+![player_moving](/assets/images/commands/states/moving.gif)
```

Comment:
[Accessibility] Alternate text should be a human readable description of the image e.g.

```suggestion
![A player walking across grass](/assets/images/commands/states/moving.gif)
```

If the image is decorative (does not contain vital information) then the alternate text may be left empty.

Reply:
I see, since the images are just decorative, I'll remove the alt texts ^^